### DISCUSSION TITLE
Windows connections speed reach zero b/s every 27-30 seconds

### DISCUSSION THREAD

**Discussion by IoSonoPiero** [localsend/localsend#539] [Discussion (English, International)]
Hello people, I hope I can explain myself.
After installing localsend on my Windows 11 22H2, I suddenly noticed that my connection dropped to 0 after about 27-30 seconds.
I mean: if I'm downloading a file, I see the connection stops, then after 4 seconds, it returns to max.
If I close the app, the issue disappears.
I made a lot of attempts to understand if other components are interfering with localsend (antivirus, background apps, etc.).
I know it's bizarre, but I have no connection drops if I avoid running localsend at Windows startup.
I'm on Wi-Fi, no ETH. The Wi-Fi is always connected; for connection drop, I mean the speed reaches 0, then goes to max again.

I'm ready to do tests if necessary.
Do you have any suggestions, please?
Regards,
Piero

**Comment by Tienisto** [comment]
This seems weird. Maybe it is due to autostart?

**Reply by IoSonoPiero** [reply]
Hello!
So, I turned off autostart and rebooted.
I've started Windows again.
Everything works great for me. I start localsend; I have the speed issue about every 30 seconds for about 4 seconds, and then everything returns to normal.
This happens right now while I'm writing this, too.
